CppCoreGuidelines.md

```diff
@@ -347,7 +347,7 @@ Philosophy rules summary:
 
 Philosophical rules are generally not mechanically checkable.
 However, individual rules reflecting these philosophical themes are.
-Without a philosophical basis the more concrete/specific/checkable rules lack rationale.
+Philosophical rules serve as a basis and rationale for the concrete/specific/checkable rules.
 
```

Comment:
The meaning of those two sentences is not the same.
In the new one, "philosophical rules" are both the basis and the rationale.
In the original, they are just the basis.